src/v2i-hub/ERVCloudForwardingPlugin/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+PROJECT(ERVCloudForwardingPlugin VERSION 7.3.1 LANGUAGES CXX)
+
+SET(TMX_PLUGIN_NAME "ERVCloudForwarding")
+add_compile_options(-fPIC)
+
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Network REQUIRED)
+find_package(CURL REQUIRED)
```

Comment:
Is there a reason we are using Qt core our REST API server and then CURL for our REST client. Not sure this is the correct place to address this but we would like to start consolidate the libraries we use. Will leave it up to @dan-du-car if we think tackling this issue as part of this PR is appropriate or we should try to improve this in the future.

Reply:
I would leave it for future improvement and create a story/issue specifically to address that if needed.

Reply:
Paul, QT is not for client applications, just server.  I think the real consolidation we should make at some point is removing the use of the curl executable and using the C++ library for that.  The CURL library is already used the CARMACloud plugin.  I think this is fine as is.

Reply:
For the record QT has both as Client and Server functionality. We have used both in CARMA-Streets and in V2X-Hub (See Port Drayage Plugin). The one benefit of using QT is that some OpenAPI codgen tools support generation of QT REST Clients and Servers based on an OpenAPI api description doc. If we do not want to worry about this here then that is fine although every additional implementation we make of a REST Client is more work for us to clean up in the future. We have also used CURL in multiple places in V2X-Hub and it seems to be less lines of code so at some point we need to make a decision and move from there